connectivity/FEATURE_BLE/include/ble/SecurityManager.h

```diff
@@ -766,6 +781,7 @@ class SecurityManager
      */
     ble_error_t setOOBDataUsage(ble::connection_handle_t connectionHandle, bool useOOB, bool OOBProvidesMITM = true);
 
+#if BLE_FEATURE_SECURE_CONNECTIONS
```

Comment:
is SC required for this? same for notification below

Reply:
Numeric comparison is reserved to secure connection (that's the confirm part). Keypress events are reserved for secure connection too.